src/linksbagmanager.cpp

```diff
@@ -228,6 +228,7 @@ void LinksBagManager::loadBookmarksFromCache()
         settings.setArrayIndex(i);
         QByteArray data = settings.value("SerializedData").toByteArray();
         Bookmark bm = Bookmark::Deserialize(data);
+        qDebug() << "no nie" << bm.GetContent() << "czemu to nie dziala";
```

Comment:
in english, please

Reply:
omg, sorry, I forgot to remove that line